README.md

```diff
@@ -45,23 +47,16 @@ To grant access:
+1. Azure.Identity - makes it easy to fetch access tokens for Service-to-Azure-Service authentication scenarios.
+2. Azure.Messaging.ServiceBus - contains methods for interacting with Service Bus. 
 
-```csharp    
-var tokenProvider = TokenProvider.CreateManagedServiceIdentityTokenProvider();
-QueueClient sendClient = new QueueClient($"sb://{Config.Namespace}.servicebus.windows.net/", Config.Queue, tokenProvider);
-await sendClient.SendAsync(new Message(Encoding.UTF8.GetBytes(messageInfo.MessageToSend)));
-await sendClient.CloseAsync();
-```
+The relevant code is in WebAppServiceBus/WebAppServiceBus/Controllers/HomeController.cs file. In the constructor for HomeController, a ServiceBusClient instance is created using the DefaultAzureCredential. A ServiceBusProcessor is started to start receiving messages from our queue. When the **Send** method is invoked, a message is sent to the queue. When the **Receive** method is invoked, a summary of the messages that have been received is displayed on the web page.
 
 ## Step 4: Update the Service Bus Namespace and Service Bus Queue names
 In the Web.config file, change the Service Bus Namespace and Queue to the ones you just created. Replace **ServiceBusNamespace** with the name of your Service Bus Namespace and **ServiceBusQueue** with the name of your Service Bus Queue.
 
 ## Step 5: Run the application on your local development machine
-When running your sample, the previously-configured QueueClient will use the ManagedServiceIdentityTokenProvider, which uses the developer's security context to get a token to authenticate to Service Bus. This removes the need to create a Service Bus shared access key and share it with the development team. It also prevents credentials from being checked in to source code. For local development, the ManagedServiceIdentityTokenProvider will use **Visual Studio**, **Azure CLI**, or **Active Directory Integrated Authentication** to authenticate to Azure AD to get a token. That token will be used to both send and receive data from your Service Bus Queue.
+When running your sample, the previously-configured ServiceBusClient will use the DefaultAzureCredential, which uses the developer's security context to get a token to authenticate to Service Bus. This removes the need to create a Service Bus shared access key and share it with the development team. It also prevents credentials from being checked in to source code. For local development, the DefaultAzureCredential will use **Visual Studio**, **Azure CLI**, or **Active Directory Integrated Authentication** to authenticate to Azure AD to get a token. That token will be used to both send and receive data from your Service Bus Queue.
```

Comment:
The DefaultAzureCredential has a different order and set of auth providers it tries from AppAuth so this should be reworded to account for that, or at least the last sentence since that list is specific to the track 1 AppAuth. This has the info for defaultazurecredential: https://docs.microsoft.com/en-us/dotnet/api/azure.identity.defaultazurecredential?view=azure-dotnet

Reply:
We already link to the DefaultAzureCredential docs. We can probably just remove that line about local development.

Reply:
Yup my thoughts as well, I just removed in subsequent commit.

Reply:
Thank you!